apis/hashicups/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
How are these added?

Reply:
I was using go1.17. Switched to 1.16 to remove that extra build constraint.